_search-plugins/knn/knn-index.md

```diff
@@ -197,6 +231,6 @@ Setting | Default | Updateable | Description
 :--- | :--- | :--- | :---
 `index.knn` | false | false | Whether the index should build native library indices for the `knn_vector` fields. If set to false, the `knn_vector` fields will be stored in doc values, but Approximate k-NN search functionality will be disabled.
```

Comment:
There is an instance of "indices" here. Also, ensure that we either consistently capitalize or lowercase "approximate" when it precedes "k-NN search" throughout.